README.md

```diff
@@ -1,5 +1,16 @@
-Syntax highlightning for Spicy
-==============================
+Syntax highlighting for Spicy
+=============================
 
 This repository contains Vim syntax highlighting for
 [Spicy](https://docs.zeek.org/projects/spicy/en/latest/).
+
+Formatting
+----------
+
+If using [ALE](https://github.com/dense-analysis/ale)
+and [spicy-format](https://github.com/bbannier/spicy-format) is available
+in your path, add the following settings to ``~/.vimrc`` or
+``~/.vim/ftplugin/spicy.vim`` for formatting on save:
+
+    let g:ale_fixers.spicy = ['spicy-format']
```

Comment:
> Since this just associates a formatter with a ftype, let's instead add this always in after/ftplugin/spicy.vim,

This also enables it for `:ALEFix` which I guess is fine, even if a bit forced upon maybe? Though, there's not effect unless `g:ale_fix_on_save = 1` is set.

Reply:
Yes, that seems like a reasonable default, and also what upstream seems to suggest, https://github.com/dense-analysis/ale?tab=readme-ov-file#fixing.